### ISSUE TITLE
strange result of linear combinations mode

### ISSUE THREAD

**Issue by ChiFang** [LinHungShi/GCNetwork#15]
In order to make final layer (soft argmin) don't limits the output value.
I use linear combinations to predict. (my result is good with your pre-trained model in softargmin)

my hyperparams:
{
	"max_disp": 64,
	"base_num_filters": 32,
	"first_kernel_size": 5, 
	"kernel_size": 3,
	"num_res": 8, 
	"num_down_conv": 4,
	"resnet": 1,
	"output": "linear",
	"act_func": "relu",
	"h_act_func": "sigmoid",
	"ds_stride": 2, 
	"padding": "same",
	"data_format": "channels_last"
}

but the result is very strange. I  print out the shape and min max

Shape:  (1, 320, 320)
Min:  -37094.2 Max 4468.41
Shape After squeeze:  (320, 320)

It seems the output range  of GCNetwork is wrong....

Any suggestion?
Thanks,
Fang
